torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -2211,7 +2222,7 @@ def generate_py_arg_inner(lines, raw_arg, arg_type):
                 )
                 return f"PyLong_FromLongLong({self.expr_printer(expr)})"
             elif isinstance(arg_type, torch.FloatType):
```

Comment:
@desertfire would we get assertion error because arg_type is torch.FloatType instead of float?

Reply:
`torch.FloatType` is the type info from an op schema, `val` should still be float here.